pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -88,22 +88,18 @@ class ModificationsExportDialog extends React.Component {
                    title={_("Automation script") }>
                 <Tabs activeKey={this.state.active_tab} onSelect={this.handleSelect}>
                     <Tab eventKey="shell" title={_("Shell script")}>
-                        <pre>
-                            {this.props.shell.trim()}
-                        </pre>
+                        <TextArea resizeOrientation='vertical' isReadOnly defaultValue={this.props.shell.trim()} />
                     </Tab>
                     <Tab eventKey="ansible" title={_("Ansible")}>
-                        <pre>
-                            {this.props.ansible.trim()}
-                        </pre>
-                        <div>
-                            <span className="fa fa-question-circle fa-xs" />
+                        <TextArea resizeOrientation='vertical' isReadOnly defaultValue={this.props.ansible.trim()} />
+                        <div className="ansible-docs-link">
+                            <QuestionCircleIcon size="xs" />
                             { _("Create new task file with this content.") }
-                            <a href="https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html"
-                                target="_blank" rel="noopener noreferrer">
-                                <i className="fa fa-external-link fa-xs" />
+                            <Button variant="link" component="a" href="https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html"
+                                    target="_blank"
+                                    icon={<ExternalLinkSquareAltIcon />}>
```

Comment:
I find this icons a bit weird. We use `ExternalLinkAltIcon` elsewhere, I would go with that.

Reply:
I am fine with both. For the record I just picked it because of the example here: https://www.patternfly.org/v4/components/button/#variations